src/sentry/integrations/bitbucket/issues.py

```diff
@@ -12,12 +12,6 @@
     ('blocker', 'Blocker'),
 )
 
-ERR_404 = (
-    'Bitbucket returned a 404. Please make sure that '
-    'the repo exists, you have access to it, and it has '
-    'issue tracking enabled.'
```

Comment:
I removed this because the BB error that comes back is explicit about issue tracking being disabled.